gui/src/pages/inventory.tsx

```diff
@@ -67,6 +67,31 @@ export default function Inventory() {
   useWebviewListener("toggleOverlay", () => handleTabChange("inventory"), []);
   useWebviewListener("getCurrentTab", async () => activeTab, [activeTab]);
 
+  useEffect(() => {
+    const handleKeyDown = (e: KeyboardEventInit) => {
+      // Check if Ctrl/Cmd + Tab is pressed
```

Comment:
we should not change cmd+tab behavior ! that is a system thing, and we should keep it

Reply:
instead, cmd+e should be the equivalent of cmd+tab but within pearai,

what i mean is that you hold cmd , press e, it opens it with the first one selected. press it again (while holding cmd) and it goes to the next selection.

next stage of this (another pr) would have a sorting of most recently used tool.